activiti-engine/src/main/java/org/activiti/engine/delegate/event/impl/ActivitiMessageEventImpl.java

```diff
@@ -24,7 +24,9 @@
 public class ActivitiMessageEventImpl extends ActivitiActivityEventImpl implements ActivitiMessageEvent {
 
   protected String messageName;
+  protected String correlationKey;
```

Comment:
I didn't want change the existing semantics for new attributes. It would make sense to change all existing attributes to private if we do this for new attributes.